messaging/src/main/scala/uk/ac/wellcome/messaging/worker/monitoring/MonitoringClient.scala

```diff
@@ -0,0 +1,8 @@
+package uk.ac.wellcome.messaging.worker.monitoring
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait MonitoringClient {
```

Comment:
We already have a monitoring lib, so eventually I'd like to see this move there.

And you'll also add a `CloudWatchMonitoringClient` class as a primary use case?

Reply:
Yes.